hs/test/Main.hs

```diff
@@ -185,19 +199,21 @@ goldenTests mkTest ext subdir = do
 
 main :: IO ()
 main = do
-  parseTests <- testsFor (Proxy @ParseErr) parseErrExample ".rsh" "parse-errors"
-  compileTests <- testsFor (Proxy @CompileErr) compileErrExample ".rsh" "compile-errors"
+  parseTests <- testsFor (Proxy :: Proxy ParseErr) parseErrExample ".rsh" "parse-errors"
+  compileTests <- testsFor (Proxy :: Proxy CompileErr) compileErrExample ".rsh" "compile-errors"
```

Comment:
This was the only part of the code I had to change to accommodate ormolu. It apparently doesn't understand type application syntax using `@`.